examples/complete-alb/main.tf

```
@@ -563,9 +488,8 @@ module "vpc" {
   private_subnets = [for k, v in local.azs : cidrsubnet(local.vpc_cidr, 4, k)]
   public_subnets  = [for k, v in local.azs : cidrsubnet(local.vpc_cidr, 8, k + 48)]
 
-  # Disabled NAT gateway to save a few seconds running this example
-  enable_nat_gateway   = false
-  enable_dns_hostnames = true
+  enable_nat_gateway = true
```

```suggestion
  enable_nat_gateway = false
```

I believe that we don't need NAT in examples.

---

fixed in https://github.com/terraform-aws-modules/terraform-aws-alb/pull/305/commits/522c07009fa3cc79200886990f3ab8ba1ac76231